test/JDBC/expected/datatype_test.out

```diff
@@ -122,8 +126,8 @@ GO
 SELECT * FROM datatype_test_t1;
 GO
 ~~START~~
-bigint#!#binary#!#bit#!#char#!#date#!#datetime#!#datetime2#!#datetimeoffset#!#numeric#!#float#!#image#!#int#!#money#!#nchar#!#ntext#!#numeric#!#nvarchar#!#real#!#smalldatetime#!#smallint#!#smallmoney#!#sql_variant#!#varchar#!#text#!#time#!#tinyint#!#uniqueidentifier#!#varbinary#!#varchar#!#xml
-1#!#1A00000000000000000000000000000000000000#!#1#!#abc                 #!#2000-06-01#!#2000-06-01 00:00:00.0#!#2000-06-01 00:00:00.0000000#!#2000-06-01 00:00:00.0000000 +00:00#!#8.4230#!#1.43E19#!#0456#!#1#!#50.0000#!#😊😋😎😍😅😆        #!#😊😋😎😍😅😆#!#8.4230#!#😊😋😎😍😅😆#!#1.43E19#!#2000-06-01 00:00:00.0#!#1#!#50.0000#!#123#!#abc#!#abc#!#01:01:12.0000000#!#1#!#1E984725-C51C-4BF4-9960-E1C80E27ABA0#!#01AB#!#abc#!#<xml></xml>
+bigint#!#binary#!#bit#!#char#!#date#!#datetime#!#datetime2#!#datetimeoffset#!#numeric#!#float#!#image#!#int#!#money#!#nchar#!#ntext#!#numeric#!#nvarchar#!#real#!#smalldatetime#!#smallint#!#smallmoney#!#sql_variant#!#varchar#!#text#!#time#!#tinyint#!#uniqueidentifier#!#varbinary#!#varchar#!#xml#!#geometry#!#geography
+1#!#1A00000000000000000000000000000000000000#!#1#!#abc                 #!#2000-06-01#!#2000-06-01 00:00:00.0#!#2000-06-01 00:00:00.0000000#!#2000-06-01 00:00:00.0000000 +00:00#!#8.4230#!#1.43E19#!#0456#!#1#!#50.0000#!#😊😋😎😍😅😆        #!#😊😋😎😍😅😆#!#8.4230#!#😊😋😎😍😅😆#!#1.43E19#!#2000-06-01 00:00:00.0#!#1#!#50.0000#!#123#!#abc#!#abc#!#01:01:12.0000000#!#1#!#1E984725-C51C-4BF4-9960-E1C80E27ABA0#!#01AB#!#abc#!#<xml></xml>#!#E6100000010C00000000000000000000000000000000#!#E6100000010C00000000000000000000000000000000
```

Comment:
Let's print geography and geometry columns as text (STAsText)

Reply:
Done